Tasks/AzureCLI/azureclitask.ts

```diff
@@ -44,13 +54,18 @@ export class azureclitask {
             tl.mkdirP(cwd);
             tl.cd(cwd);
 
+            if (scriptType === "bash") {
+                tool.pathArg(scriptPath);
+            }
+            else {
+                tool = tl.createToolRunner(tl.which(scriptPath, true));
```

Comment:
Add a package.json and use vsts-task-lib@0.9.x (latest).  See shelljs how we did this.

That function is now tl.tool();